lib/api.js

```diff
@@ -205,18 +267,31 @@ NodeCG.sendMessageToBundle = function (messageName, bundleName, data, cb) {
 			data = null;
 		}
 
-		window.socket.emit('message', {
-			bundleName,
-			messageName,
-			content: data
-		}, cb);
-	} else {
-		io.emit('message', {
-			bundleName,
-			messageName,
-			content: data
+		return new Promise((resolve, reject) => {
+			window.socket.emit('message', {
+				bundleName,
+				messageName,
+				content: data
+			}, (err, ...args) => {
+				if (typeof cb === 'function') {
+					cb(err, ...args);
+					return;
```

Comment:
Nice.
So this doesn't spit out UnhandledPromiseRejection notices, `return` just satisfies the promise resolution?

Reply:
It doesn't resolve nor reject the promise, which to my knowledge is totally okay.

Reply:
Maybe I should modify this to just not even return a promise if `cb` is a function.

Reply:
Pushed that change.